src/utils/CryptoUtils.ts

```diff
@@ -29,4 +37,17 @@ export class CryptoUtils {
         const decipher = crypto.createDecipheriv(this.algorithm, key, Buffer.from(iv, "hex"));
         return decipher.update(encrypted, 'hex', 'utf8') + decipher.final('utf8');
     }
+
+    decodeAuthToken(authToken: string): any {
```

Comment:
```suggestion
    decodeAuthToken(authToken: string): any {
```

Let's specify the function return type